### ISSUE TITLE
How do I open 3 tabs at start

### ISSUE THREAD

**Issue by csdvrx** [wez/wezterm#3646]
# Section of the documentation

[SpawnCommand](https://wezfurlong.org/wezterm/config/lua/SpawnCommand.html)

**Issue**

I would like each window to open 3 tabs by default, say  one running bash, the other busybox ash, and the last one top

It's not clear at all how to do that with the config file without using panes: I'd just like to open multiple tabs by default.

With the commandline, I can just type `wezterm cli spawn`

I suppose it would entail using wezterm.on('gui-startup', function(cmd)) but I'm not sure how.

**Comment by devmatteini** [comment]
Yes you need to use [gui-startup](https://wezfurlong.org/wezterm/config/lua/gui-events/gui-startup.html).

If you follow the documentation examples, you can create something like this in `wezterm.lua`:

```lua
local wezterm = require "wezterm"
local mux = wezterm.mux

wezterm.on('gui-startup', function()
  local _, first_pane, window = mux.spawn_window {}
  local _, second_pane, _ = window:spawn_tab {}
  local _, third_pane, _ = window:spawn_tab {}

  second_pane:send_text "<insert busybox command>\n"
  third_pane:send_text "top\n"
  -- '\n' this will execute you shell command
end)
```

**Comment by csdvrx** [comment]
Wonderful example, thank you!

I would suggest to add that to the documentation, but with a tweak: the send_text didn't work for me because of delays in the shell start.

I know there's command ands args {}, they might be better suited.
